.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Feature request
+about: Suggest an improvement to vcpkg or its packages
+title: ''
+labels: ''
```

Comment:
Maybe add a "features - needs triage" label or something so that it's easy to find feature requests we can then triage into port or vcpkg requests.